pkg/apis/networking/types.go

```diff
@@ -55,6 +55,16 @@ type NetworkPolicySpec struct {
 	// solely to ensure that the pods it selects are isolated by default)
 	// +optional
 	Ingress []NetworkPolicyIngressRule
+
+	// List of egress rules to be applied to the selected pods. Outgoing traffic is
+	//allowed unbounded if there are no NetworkPolicies selecting the pod
```

Comment:
missing space after "//", and the word "unbounded" doesn't seem necessary, and also we need to add the thing discussed in https://groups.google.com/d/msg/kubernetes-sig-network/nWyAL-8vKV8/DDRZ9Jx8BwAJ so that all pods with ingress rules don't suddenly become default-deny to egress